### ISSUE TITLE
[Fluxstore Woocommerce] -  Firebase Setup 

### ISSUE THREAD

**Issue by gugan7** [inspireui/support#2591]
> - Thank you for reaching please follow the format below to create a new ticket, please explain as more detail as possible and allow 24 - 48 hours to receive an answer to your ticket.
> - Try to explain your all problems in one ticket, instead of creating multi tickets.
> - After creating an issue, please take the screenshot of http://codecanyon.net/download - to let us know you have purchased our product, or quick note as a comment that you have created this ticket, example: https://tppr.me/DJA8L
> - IMPORTANT: We will close the ticket if you could not prove as our customer.

---------

1. My issue:
Hi @trunghieuvn and @minhcasi ,

The documentation video on Push notification is not clear. https://docs.inspireui.com/fluxstore/firebase/#2-push-notifications-via-firebase

The part where push notification function added is not clear. From the video a existing Firebase folder was used but never tell from where to get the Firebase folder. 

Any way I tried to proceed without it. Once i install firebase cli , i tried firebase init functions , it then automatically authenticate to firebase and download dependency for it. The file .firebaserc was automatically generated with correct project-id. Then finally i did firebase deploy. Below is the output . I go back to Firebase console under Functions but couldn't find any functions there. Do we need to add something to index.js ? if yes , please let us know what is code to put there.

`MFBM-2W80WCHV2H:source guganesanselvaraj$ firebase deploy

=== Deploying to 'nextcommerce-367eb'...

i deploying functions
i functions: ensuring necessary APIs are enabled...
✔ functions: all necessary APIs are enabled
i functions: preparing functions directory for uploading...

✔ Deploy complete!

2. Screenshots issues (required):

3. Proof of my purchased (required):
I have valid proof of purchase in case https://github.com/inspireui/support/issues/2494
3. Config files (required)

**Comment by minhcasi** [comment]
@gugan7 - we will upload a new video for Firebase push notification again :)

**Comment by gugan7** [comment]
Hi @minhcasi ,

Please let me know asap once the video is ready. Our development task is pending because of this. 

Thank you

**Comment by minhcasi** [comment]
@trunghieuvn - pls help to update the video ASAP

**Comment by gugan7** [comment]
Hi ,
When is the ETA ? if its going to take long time to create the video , can you just upload the firebase file ?

**Comment by minhcasi** [comment]
@gugan7 - we are working on this video, it will be available ASAP - thanks

**Comment by trunghieuvn** [comment]
@gugan7 please follow tutorial here: https://github.com/inspireui/support/wiki/%5BFeature%5D-Setup-firebase-cloud-messaging(FCM)-push-notification-in-IOS
 i will update video and send for you later
thanks

**Comment by minhcasi** [comment]
@gugan7 - https://www.youtube.com/watch?v=rFCcWNRPrSk please help to check this detail video - thank you!
